ci/pipeline.properties

```diff
@@ -11,6 +11,7 @@ docker.mongodb.4.4.version=4.4.25
 docker.mongodb.5.0.version=5.0.21
 docker.mongodb.6.0.version=6.0.10
 docker.mongodb.7.0.version=7.0.2
+docker.mongodb.8.0.version=8.0.0
```

Comment:
We need to introduce that property in https://github.com/spring-projects/spring-data-build/blob/main/ci/pipeline.properties as well.

Reply:
yes, once this is approved, I'll add it to data-build.